src/main/java/org/broadinstitute/hellbender/tools/exome/allelefraction/AFData.java

```diff
@@ -0,0 +1,77 @@
+package org.broadinstitute.hellbender.tools.exome.allelefraction;
+
+import org.apache.commons.lang.math.IntRange;
+import org.broadinstitute.hellbender.tools.exome.AllelicCount;
+import org.broadinstitute.hellbender.tools.exome.SegmentedModel;
+import org.broadinstitute.hellbender.tools.exome.TargetCollection;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.mcmc.DataCollection;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+/**
+ *
+ * Created by davidben on 11/10/15.
+ */
+public final class AFData implements DataCollection {
+    // allelic counts indexed by het
```

Comment:
@samuelklee In my pedantic opinion I try (and often fail to live up to my own pedantry) to use "SNP" to mean a site that is polymorphic *in the population* and "het" to mean a locus at which *an individual* is heterozygous.  Eg "`HetPulldown` searches for hets in the normal sample from a list of known SNPs."  Perhaps @vdauwera could weigh in on whether / how this distinction is made elsewhere in the GATK?

By the way, I agree that we should be consistent, and I'm happy to take on the task of tidying.  Which previous usage are you referring to?